src/m365/aad/commands/o365group/o365group-add.ts

```diff
@@ -179,6 +138,95 @@ class AadO365GroupAddCommand extends GraphCommand {
       }, (rawRes: any): void => this.handleRejectedODataJsonPromise(rawRes, logger, cb));
   }
 
+
+  private validateOwners(logger: Logger, args: CommandArgs): Promise<any> {
+    if (!args.options.owners) {
+      if (this.debug) {
+        logger.logToStderr('Owners not set. Skipping');
+      }
+      return Promise.resolve(undefined as any);
+    }
+
+    if (this.verbose) {
+      logger.logToStderr('Retrieving user information to set group owners...');
+    }
+
+    const owners: string[] = args.options.owners.split(',').map(o => o.trim());
+    const promises: any[] = [];
+    let ownerIds: any[] = [];
+
+    owners.forEach(owner => {
+      const requestOptions: any = {
+        url: `${this.resource}/v1.0/users?$filter=${`userPrincipalName eq '${owner}'`}&$select=id,userPrincipalName`,
+        headers: {
+          'content-type': 'application/json'
+        },
+        responseType: 'json'
+      };
+
+      promises.push(request.get(requestOptions));
+    });
+
+    return Promise.all(promises).then((ownersRes: { value: { id: string; userPrincipalName: string }[] }[]): Promise<any> => {
```

Comment:
Let's not use `Promise<any>` as the return type. Since we know which type of data we're returning, let's use it instead to avoid refactoring errors in the future.